README.NewResourceFromTerraform.md

```diff
@@ -615,6 +615,11 @@ update to date, we need to add the new CRDs and samples to existing doc.
     `https://cloud.google.com` (i.e.
     `https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances`).
     Add an additional paragraph/table if needed.
+1.  Update the URL in "Before you begin" section, points to the "Prerequisites" or 
```

Comment:
It's unclear to me that:

Do we _require_ resource contributors to add this section, or do we _suggest_ resource contributors add this section when the resource hit any conditions we set?

With the fact that only one resource has this section, I don't think we require resource contributors to do so.

I'd probably update the wording a bit to something like (I didn't use the md format in the text suggested below so please adjust it accordingly):

If additional setup is necessary before the newly added resource can be applied successfully, and the setup can't be done via the Config Connector resources or features, add a "Before You Begin" section. E.g.  https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/b019cb7c3e1dea745b804d9c6d8cb6f97fa67996/scripts/generate-google3-docs/resource-reference/generated/resource-docs/containerattached/containerattachedcluster.md